code/modules/projectiles/gun_attachables.dm

```diff
@@ -2073,6 +2073,14 @@ Defined in conflicts.dm of the #defines folder.
 	burn_level = BURN_LEVEL_TIER_5
 	burn_duration = BURN_TIME_TIER_2
 
+/obj/item/attachable/attached_gun/flamer/advanced//essentially a better version of mini-flamer for PMCs, copy of integrated but is removable
```

Comment:
if it's the same as the integrated flamethrower it should be a subtype. also it should only be added for pmcs, survivors should not be able to find these

Reply:
Oh yeah I forgot to do that, will do it now.
👍